en/docs/develop/product-apis/admin-apis/admin-v1/admin-v1.yaml

```diff
@@ -2617,6 +2906,19 @@ paths:
           required: true
           schema:
             $ref: '#/definitions/Label'
+          examples:
+            test:
+              summary: summary1
```

Comment:
Why do we have to repeat the example here? Can't we move this to label definition itself?

Reply:
this was a typo. Fixed with https://github.com/wso2/docs-apim/pull/1842/commits/09f97e448ff264ecc0b6025cb1c9ac19f2228951